scripts/preprocess-health.py

```
@@ -0,0 +1,425 @@
+
+    if len(positional) < 2:
+        print("Usage: python preprocess-health.py <export.zip|export.xml> <output.json> [--max-hr=185]")
+        sys.exit(1)
+
+    input_path, output_path = positional[0], positional[1]
+    if not os.path.exists(input_path):
+        raise SystemExit(f"  Input not found: {input_path}")
+
+    file_size = os.path.getsize(input_path) / 1024 / 1024
+    print(f"\n{'=' * 50}")
+    print(f"  Apple Health → courtfolio cardio.json")
```

Ruff (0.15.11)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)